extensions/sql-migration/src/wizard/skuRecommendationPage.ts

```diff
@@ -607,7 +607,12 @@ export class SKURecommendationPage extends MigrationWizardPage {
 
 	private createAssessmentProgress(): azdata.FlexContainer {
 
-		this._assessmentLoader = this._view.modelBuilder.loadingComponent().component();
+		this._assessmentLoader = this._view.modelBuilder.loadingComponent().withProps({
+			CSSStyles: {
+				'height': '60px'
```

Comment:
The loading component already has a height property so I would think it would make more sense to actually use that instead of having people go through CSSStyles

Reply:
Although I just looked at your screenshots and it seems like height here is just being used to align the component which seems a little odd. Where did you get 60px from? If you're trying to align stuff in a container generally I'd expect the use of styles such as alignItems.

Anyways I don't really have a huge issue with adding the styles here, just curious how you arrived at this solution.

Reply:
I will change this to 100% to make it match the container height. I can also use margins to center align but matching the container height seems like the better solution.

Reply:
Should I add a new property called loaderIconCSSStyles to change loader icon css props?

Reply:
No, that's a standard property we use elsewhere so no reason to add a new one.

Although did you try setting the height style when you add it to the parent container instead? That's generally the way I've seen people solve alignment stuff.

Reply:
I tried that but it did not make any difference to the actual component.